blockchain/emulator_test.go

```diff
@@ -59,5 +63,123 @@ func Test_TranslateAddress(t *testing.T) {
 		out := translateAddresses(in[0])
 		assert.Equal(t, string(in[1]), string(out))
 	}
+}
+
+// Test_NewEmulator tests creating a large number of new accounts and validates corresponding storage addresses
+func Test_NewEmulator(t *testing.T) {
+	emu, err := newEmulator()
+	assert.NoError(t, err)
+
+	var accountList []*flowsdk.Account
+
+	const testAccounts int = 1000
+
+	fmt.Println("Creating", testAccounts, "new accounts...")
```

Comment:
Avoid outputting during tests.
```suggestion
	fmt.Println("Creating", testAccounts, "new accounts...")
```